File: OracleSqlTestInPython/Relationship.py
# Base class for Relationship.
class Relationship(object):
    """description of class"""

    #region Fields

    # The id field of the Relationship class.
    __id = None

    # The name field of the Relationship class.
    __name = None

    # The isDeleted field of the Relationship class.
    __isDeleted = None

    #endregion Fields

    #region Properties

    # Gets the id.
    def getId(self):
        return self.__id

    # Sets the id.
    def setId(self, value):
        self.__id = value

    # Gets the name.
    def getName(self):
        return self.__name

    # Sets the name.
    def setName(self, value):
        self.__name = value

    # Gets the isDeleted.
    def getIsDeleted(self):
        return self.__isDeleted

    # Sets the isDeleted.
    def setIsDeleted(self, value):
        self.__isDeleted = value

    #endregion Properties

    #region Constructors

    # Initializes a new instance of the Relationship class.
    def __init__(self):
        self.__id = None
        self.__name = None
        self.__isDeleted = None

    # Initializes a new instance of the Relationship class.
    def __init__(self, id, name, isdeleted):
        self.__id = id
        self.__name = name
        self.__isDeleted = isdeleted

    #endregion Constructors

    #region Methods

    #endregion Methods

File: MsSqlTestInLua/Relationship.lua
-- Base class for Relationship.
Relationship = {__id = 0, __name = "", __isDeleted = false}

-- Properties

-- Gets the id.
function Relationship : getId()
   return self.__id
end

-- Sets the id.
function Relationship : setId(value)
   self.__id = value
end

-- Gets the name.
function Relationship : getName()
   return self.__name
end

-- Sets the name.
function Relationship : setName(value)
   self.__name = value
end

-- Gets the isDeleted.
function Relationship : getIsDeleted()
   return self.__isDeleted
end

-- Sets the isDeleted.
function Relationship : setIsDeleted(value)
   self.__isDeleted = value
end

-- Properties

-- Constructors

-- Initializes a new instance of the Relationship class.
function Relationship : new(o)
   o = o or {}
   setmetatable(o, self)
   self.__index = self
   
   return o
end

-- Initializes a new instance of the Relationship class.
function Relationship : new(o, id, name, isdeleted)
   o = o or {}
   setmetatable(o, self)
   self.__index = self
   self.__id = id or 0
   self.__name = name or ""
   self.__isDeleted = isdeleted or false
   
   return o
end

-- Constructors